### ISSUE TITLE
HPOS sends new registration email on each reboot

### ISSUE THREAD

**Issue by peeech** [Holo-Host/holo-nixpkgs#328]
**Description**
Once HPOS successfully registers to Holo network user receives a confirmation email with link. The same email gets sent to the user every time HPOS reboots. Checked both in qemu and on actual hardware.

**To reproduce**
Register your HPOS (that follows staging branch, commit `47e68fb` ATM) by following registration procedure. After successful registration (confirmed by email and working HP-Admin link) reboot your HPOS and wait for another registration email.

**Desired behavior**
Registration email should be triggered only once at actual HPOS registration.

**Comment by alastairong** [comment]
Possibly related: 

Jamison has confirmed that all 3 of his HoloPorts / HP+ resent confirmation emails upon auto-updating to `staging`.

P.S. For future - should I assume everything is a new issue or is it ok to post things that look related in this way?

**Comment by yegortimoshenko** [comment]
@peeech Please CC me on issues you want me to take a look at next time :)

> P.S. For future - should I assume everything is a new issue or is it ok to post things that look related in this way?

It is OK. We can always move something out to a new issue if it indeed is.

**Comment by yegortimoshenko** [comment]
I've converted this issue into a ZenHub epic and added Holo-Host/holo-auth#3 as the only issue that needs to be resolved in order to complete it. Please use ZenHub to see all of this metadata.

**Comment by yegortimoshenko** [comment]
Resolved in #323.
